src/System.Private.CoreLib/shared/System/ThrowHelper.cs

```diff
@@ -969,6 +969,7 @@ internal enum ExceptionResource
         Arg_RanksAndBounds,
         InvalidOperation_IComparerFailed,
         NotSupported_FixedSizeCollection,
+        NotSupported_SpanOverlappedOperation,
```

Comment:
It is not sufficient to add it just here. This would need to be added on mutiple other places. 

(This It won't matter once you fix the other comment about not using ThrowHelper.)

Reply:
> (This It won't matter once you fix the other comment about not using ThrowHelper.)

Ok but can you explain why I would update also "other places"?Do you mean inside vm(unmanaged)?

Reply:
No, inside this file. E.g. look for all places where `NotSupported_FixedSizeCollection` is in this file.

Reply:
Ah simply... following code path ended here, btw I'll remove, thanks Jan!

Reply:
EDIT: understood...I missed a piece.